src/main/java/seedu/storemando/model/expirydate/ItemExpiringPredicate.java

```diff
@@ -18,7 +18,7 @@
      * @param numOfDays  The input number used to filter the expiry date
      */
     public ItemExpiringPredicate(Long numOfDays) {
-        assert numOfDays > 0 : "Input number must be a positive number";
+        assert numOfDays >= 0 : "Input number must be a positive number";
```

Comment:
Can remove this assert because now the number of days can take in any number :)

Reply:
All right will do after i merge the master to this branch! :D